server/mustache-templates/newClient.mst

```diff
@@ -30,24 +30,10 @@
                 <div class="form-group row justify-content-start row-no-gutters">
                     <label for="responderPhoneNumbers" class="col-sm-3 col-form-label">Responder Phone Numbers:</label>
```

Comment:
Can you please do some manual tests that the Dashboard still works correctly?

Reply:
Ah, yes! I forgot to include that in my test plan.